_episodes/58.md

```diff
@@ -0,0 +1,125 @@
+---
+layout: episode
+title: "58: Understanding your users with Mitzu and Trino"
+date: 2024-04-02
+tags: trino
+youtube_id: "UPDATE DATE ABOVE AS WELL"
+wistia_id: ""
+sections:
+- title: "Introduction"
+  desc: "Episode 57: Understanding your users with Mitzu and Trino"
+  time: 0:00
+- title: "Releases"
+  desc: "Trino 442, 443, 444 and other updates"
+  time:
+- title: "Guest introductions"
+  desc: "István Mészáros"
+  time:
+---
+
+## Hosts
+
+* [Cole Bowden](https://www.linkedin.com/in/cole-m-bowden), Developer Advocate
+  at [Starburst](https://starburst.io)
+* [Manfred Moser](https://www.linkedin.com/in/manfredmoser), Director of Trino
+  Community Leadership at [Starburst](https://starburst.io),
+  ([@simpligility](https://twitter.com/simpligility))
+
+## Guests
+
+* [István Mészáros](https://www.linkedin.com/in/imeszaros/), CEO and founder of
+  [Mitzu](https://www.mitzu.io/)
+
+## Trino releases
+
+[Trino 442](https://trino.io/docs/current/release/release-442.html)
+
+* Add support for configuring AWS deployment type in OpenSearch connector.
+* Fix a regression from 440 in Iceberg connector.
+
+[Trino 443](https://trino.io/docs/current/release/release-443.html)
+
+* Ensure all files are deleted when native S3 file system support is enabled,
+  and some other object storage connector improvements.
+* Add support for a custom authorization header name in Prometheus connector.
+
+[Trino 444](https://trino.io/docs/current/release/release-444.html)
+
+* Update Docker image to use Java 22 for runtime.
+* Numerous performance improvements for the Snowflake connector.
+* Add support for reading `BYTE_STREAM_SPLIT` encoding in Parquet files.
+* Add support for canned access control lists with the native S3 file system.
+
+## Other Trino news
+
+* [Trino Gateway
+  7](https://github.com/trinodb/trino-gateway/blob/main/docs/release-notes.md#trino-gateway-7-21--mar-2024)
+  shipped with a new user interface thanks to a contribution from our new
+  [Starburst Trino champion Peng
+  Wei](https://www.starburst.io/community/trino-champions/)
+* Status of the continuous integration and build setup with Apache Maven
+  improved a lot thanks to our collaboration with the new [Starburst Trino
+  champion Tamas Cservenak](https://www.starburst.io/community/trino-champions/)
+* Trino Contributor Call [recap is now
+  available](https://github.com/trinodb/trino/wiki/Contributor-meetings#trino-contributor-call-21-mar-2024)
+
+## Mitzu
```

Comment:
We can change title and content for this whole section as you like @IstvanM .. just comment or let me know on slack